sw/device/lib/dif/dif_sram_ctrl.md

```diff
@@ -0,0 +1,69 @@
+---
+title: "SRAM Controller DIF Checklist"
+---
+
+<!--
+NOTE: This is a template checklist document that is required to be copied over
+to `sw/device/lib/dif/dif_sram_ctrl.md` for a new DIF that transitions
+from L0 (Specification) to L1 (Development) stage, and updated as needed.
+Once done, please remove this comment before checking it in.
+-->
+This checklist is for [Development Stage]({{< relref "/doc/project/development_stages.md" >}}) transitions for the [SRAM Controller DIF]({{< relref "hw/ip/sram_ctrl/doc" >}}).
+All checklist items refer to the content in the [Checklist]({{< relref "/doc/project/checklist.md" >}}).
+
+
+
+Type           | Item                 | Resolution  | Note/Collaterals
+---------------|----------------------|-------------|------------------
+Implementation | [DIF_EXISTS][]       | Not Started |
```

Comment:
The definition of `DIF_EXISTS` is "`dif_<ip>.h` and, optionally, `dif_<ip>.c` exist in `sw/device/lib/dif`." so I think this should be marked as Done.

Reply:
Thanks Alex, done.